internal/pkg/service/stream/storage/level/local/writer/test/testcase/testcase.go

```diff
@@ -53,17 +52,19 @@ func (tc *WriterTestCase) Run(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 
-	// Setup logger
-	logger := log.NewDebugLogger()
-	logger.ConnectTo(testhelper.VerboseStdout())
+	d, mock := dependencies.NewMockedLocalStorageScope(t)
+	cfg := mock.TestConfig()
+
+	// Start statistics collector
+	events := writer.NewEvents()
+	collector.Start(d, events, cfg.Storage.Statistics.Collector, cfg.NodeID)
 
```

Comment:
The test is simulating pod restart, see `// Simulate pod failure, restart writer` bellow in the code.

So we must now use statistics Collector, and check statistics in etcd, because writer statistics are reset to zero, after restart.